umake/frameworks/ide.py

```diff
@@ -857,6 +857,41 @@ def post_install(self):
                         categories="Development;TextEditor;"))
 
 
+class DBeaver(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, **kwargs):
+        super().__init__(name="DBeaver", description=_("Free multi-platform database tool for developers"),
+                         only_on_archs=['i386', 'amd64'],
+                         download_page="https://dbeaver.io/download/",
```

Comment:
the download page needs to be the api github releases page: "https://api.github.com/repos/dbeaver/dbeaver/releases/latest"